[transform/scale] Unit tests failing

On Windows 10:
```
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running org.openhab.transform.scale.internal.ScaleTransformServiceTest
[ERROR] Tests run: 10, Failures: 0, Errors: 6, Skipped: 0, Time elapsed: 0.886 s <<< FAILURE! - in org.openhab.transform.scale.internal.ScaleTransformServiceTest
[ERROR] org.openhab.transform.scale.internal.ScaleTransformServiceTest.testTransformQuantityType  Time elapsed: 0.116 s  <<< ERROR!
org.openhab.core.transform.TransformationException: Could not find configuration 'scale/netatmo_aq.scale' or failed to parse it.
        at org.openhab.transform.scale.internal.ScaleTransformServiceTest.testTransformQuantityType(ScaleTransformServiceTest.java:164)

[ERROR] org.openhab.transform.scale.internal.ScaleTransformServiceTest.testEvaluationOrder  Time elapsed: 0.016 s  <<< ERROR!
org.openhab.core.transform.TransformationException: Could not find configuration 'scale/evaluationorder.scale' or failed to parse it.
        at org.openhab.transform.scale.internal.ScaleTransformServiceTest.testEvaluationOrder(ScaleTransformServiceTest.java:154)

[ERROR] org.openhab.transform.scale.internal.ScaleTransformServiceTest.testTransformAndFormat  Time elapsed: 0 s  <<< ERROR!
org.openhab.core.transform.TransformationException: Could not find configuration 'scale/netatmo_aq.scale' or failed to parse it.
        at org.openhab.transform.scale.internal.ScaleTransformServiceTest.testTransformAndFormat(ScaleTransformServiceTest.java:181)

[ERROR] org.openhab.transform.scale.internal.ScaleTransformServiceTest.testCatchNonNumericValue  Time elapsed: 0 s  <<< ERROR!
org.openhab.core.transform.TransformationException: Could not find configuration 'scale/catchnonnumeric.scale' or failed to parse it.
        at org.openhab.transform.scale.internal.ScaleTransformServiceTest.testCatchNonNumericValue(ScaleTransformServiceTest.java:173)

[ERROR] org.openhab.transform.scale.internal.ScaleTransformServiceTest.testTransformByScaleLimits  Time elapsed: 0 s  <<< ERROR!
org.openhab.core.transform.TransformationException: Could not find configuration 'scale/limits.scale' or failed to parse it.
        at org.openhab.transform.scale.internal.ScaleTransformServiceTest.testTransformByScaleLimits(ScaleTransformServiceTest.java:100)

[ERROR] org.openhab.transform.scale.internal.ScaleTransformServiceTest.testTransformByScale  Time elapsed: 0.015 s  <<< ERROR!
org.openhab.core.transform.TransformationException: Could not find configuration 'scale/humidex_de.scale' or failed to parse it.
        at org.openhab.transform.scale.internal.ScaleTransformServiceTest.testTransformByScale(ScaleTransformServiceTest.java:85)

[INFO]
[INFO] Results:
[INFO]
[ERROR] Errors:
[ERROR]   ScaleTransformServiceTest.testCatchNonNumericValue:173 ▒ Transformation Could ...
[ERROR]   ScaleTransformServiceTest.testEvaluationOrder:154 ▒ Transformation Could not f...
[ERROR]   ScaleTransformServiceTest.testTransformAndFormat:181 ▒ Transformation Could no...
[ERROR]   ScaleTransformServiceTest.testTransformByScale:85 ▒ Transformation Could not f...
[ERROR]   ScaleTransformServiceTest.testTransformByScaleLimits:100 ▒ Transformation Coul...
[ERROR]   ScaleTransformServiceTest.testTransformQuantityType:164 ▒ Transformation Could...
[INFO]
[ERROR] Tests run: 10, Failures: 0, Errors: 6, Skipped: 0
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
```

--- a/bundles/org.openhab.transform.scale/src/test/java/org/openhab/transform/scale/internal/ScaleTransformServiceTest.java
+++ b/bundles/org.openhab.transform.scale/src/test/java/org/openhab/transform/scale/internal/ScaleTransformServiceTest.java
@@ -13,8 +13,7 @@
 package org.openhab.transform.scale.internal;
 
 import static org.junit.jupiter.api.Assertions.*;
-import static org.mockito.ArgumentMatchers.anyString;
-import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.ArgumentMatchers.*;
 
 import java.io.File;
 import java.io.IOException;
@@ -80,12 +79,12 @@ public class ScaleTransformServiceTest {
     @Test
     public void testTransformByScale() throws TransformationException {
         // need to be sure we'll have the german version
-        String existingscale = "scale/humidex_de.scale";
+        String existingscale = "scale" + File.separator + "humidex_de.scale";
         String source = "10";
         String transformedResponse = processor.transform(existingscale, source);
         assertEquals("nicht wesentlich", transformedResponse);
 
-        existingscale = "scale/limits.scale";
+        existingscale = "scale" + File.separator + "limits.scale";
         source = "10";
         transformedResponse = processor.transform(existingscale, source);
         assertEquals("middle", transformedResponse);
@@ -93,7 +92,7 @@ public class ScaleTransformServiceTest {
 
     @Test
     public void testTransformByScaleLimits() throws TransformationException {
-        String existingscale = "scale/limits.scale";
+        String existingscale = "scale" + File.separator + "limits.scale";
 
         // Testing upper bound opened range
         String source = "500";
@@ -106,7 +105,7 @@ public class ScaleTransformServiceTest {
         assertEquals("low", transformedResponse);
 
         // Testing unfinite up and down range
-        existingscale = "scale/catchall.scale";
+        existingscale = "scale" + File.separator + "catchall.scale";
         source = "-10";
         transformedResponse = processor.transform(existingscale, source);
         assertEquals("catchall", transformedResponse);
@@ -116,7 +115,7 @@ public class ScaleTransformServiceTest {
     public void testTransformByScaleUndef() throws TransformationException {
         // check that for undefined/non numeric value we return empty string
         // Issue #1107
-        String existingscale = "scale/humidex_fr.scale";
+        String existingscale = "scale" + File.separator + "humidex_fr.scale";
         String source = "-";
         assertThrows(TransformationException.class, () -> processor.transform(existingscale, source));
     }
@@ -125,7 +124,7 @@ public class ScaleTransformServiceTest {
     public void testTransformByScaleErrorInBounds() throws TransformationException {
         // the tested file contains inputs that generate a conversion error of the bounds
         // of range
-        String existingscale = "scale/erroneous.scale";
+        String existingscale = "scale" + File.separator + "erroneous.scale";
         String source = "15";
         try {
             @SuppressWarnings("unused")
@@ -139,7 +138,7 @@ public class ScaleTransformServiceTest {
     @Test
     public void testTransformByScaleErrorInValue() throws TransformationException {
         // checks that an error is raised when trying to scale an erroneous value
-        String existingscale = "scale/evaluationorder.scale";
+        String existingscale = "scale" + File.separator + "evaluationorder.scale";
         String source = "azerty";
         assertThrows(TransformationException.class, () -> processor.transform(existingscale, source));
     }
@@ -147,7 +146,7 @@ public class ScaleTransformServiceTest {
     @Test
     public void testEvaluationOrder() throws TransformationException {
         // Ensures that only first matching scale as presented in the file is taken in account
-        String evaluationOrder = "scale/evaluationorder.scale";
+        String evaluationOrder = "scale" + File.separator + "evaluationorder.scale";
         // This value matches two lines of the scale file
         String source = "12";
 
@@ -158,7 +157,7 @@ public class ScaleTransformServiceTest {
     @Test
     public void testTransformQuantityType() throws TransformationException {
         QuantityType<Dimensionless> airQuality = new QuantityType<>("992 ppm");
-        String aqScaleFile = "scale/netatmo_aq.scale";
+        String aqScaleFile = "scale" + File.separator + "netatmo_aq.scale";
         String expected = "Correcte (992 ppm) !";
 
         String transformedResponse = processor.transform(aqScaleFile, airQuality.toString());
@@ -168,7 +167,7 @@ public class ScaleTransformServiceTest {
     @Test
     public void testCatchNonNumericValue() throws TransformationException {
         // checks that an error is raised when trying to scale an erroneous value
-        String existingscale = "scale/catchnonnumeric.scale";
+        String existingscale = "scale" + File.separator + "catchnonnumeric.scale";
         String source = "azerty";
         String transformedResponse = processor.transform(existingscale, source);
         assertEquals("Non Numeric", transformedResponse);
@@ -176,7 +175,7 @@ public class ScaleTransformServiceTest {
 
     @Test
     public void testTransformAndFormat() throws TransformationException {
-        String existingscale = "scale/netatmo_aq.scale";
+        String existingscale = "scale" + File.separator + "netatmo_aq.scale";
         String source = "992";
         String transformedResponse = processor.transform(existingscale, source);
         assertEquals("Correcte (992) !", transformedResponse);
@@ -184,7 +183,7 @@ public class ScaleTransformServiceTest {
 
     @Test
     public void testValueExceedsRange() throws TransformationException {
-        String existingscale = "scale/humidex.scale";
+        String existingscale = "scale" + File.separator + "humidex.scale";
         String source = "200";
         assertThrows(TransformationException.class, () -> processor.transform(existingscale, source));
     }

